base/applets/jax_applet.py

```diff
@@ -25,10 +28,52 @@ def add_labrad_ip_argument(self, applet, default_ip="127.0.0.1"):
         applet.argparser.add_argument("--ip", type=str, default=default_ip,
                                       help="LabRAD manager IP address to connect to")
 
+    @classmethod
+    def add_id_argument(self, applet, default_id=""):
+        """Adds an argument to set the ID of the applet.
+
+        Applet ID is used to assign different config files to instances of a same applet class.
+
+        Args:
+            applet: artiq.applets.simple.SimpleApplet object.
+            default_id: str, default ID of the applet. Default "".
+        """
+        applet.argparser.add_argument("--id", type=str, default=default_id,
+                                      help="Configuration ID")
+
     def __init__(self, **kwds):
         super().__init__(**kwds)
         self._labrad_loop = None
 
+    def load_config_file(self, module_name, args):
+        """Loads the config file content into self.config.
+
+        Arguments:
```

Comment:
Do we want to use Args: or Arguments: ? Just thinking about consistency with the docstring above.

Reply:
Good catch! Fixed in the newest commit.